src/nvim/getchar.c

```diff
@@ -1979,14 +1979,15 @@ static int handle_mapping(int *keylenp, bool *timedout, int *mapdepth)
       i = FAIL;
     } else {
       int noremap;
+      static char plug_map[] = { (char)K_SPECIAL, (char)KEY2TERMCAP0(K_PLUG),
+                                (char)KEY2TERMCAP1(K_PLUG), '\0' };  // refernce to match against.
```

Comment:
I feel this is a bit redundant. I'd suggest this (there is also a typo in comment):
```suggestion
      static const char plug_map[] = {K_SPECIAL, KS_EXTRA, KE_PLUG, NUL};  // reference to match against.
```

I'd also suggest the same for `K_LUA` on line 2672

Reply:
Done.